packages/esm-patient-vitals-app/src/vitals/paginated-vitals.component.tsx

```diff
@@ -39,15 +39,35 @@ const PaginatedVitals: React.FC<PaginatedVitalsProps> = ({
   const StyledTableCell = ({ interpretation, children }: { interpretation: string; children: React.ReactNode }) => {
     switch (interpretation) {
       case 'critically_high':
-        return <TableCell className={styles.criticallyHigh}>{children}</TableCell>;
+        return (
+          <TableCell className={styles.criticallyHigh}>
+            <span>{children}</span>
```

Comment:
Do we need the span or can we just apply the padding directly to the table cell here?

Reply:
@ibacher, I tried avoiding a nested element/component but I would not generate new borders except the tablecell borders. And these borders had issues with the `border-collapse`. So I needed to generate new borders for the content.

Could there be an approach I am not thinking about?. I appreciate your guidance.